docs/content/topics/operators/security.md

```diff
@@ -51,30 +58,31 @@ secrets, and persistent volume claims.
 
 ## Project Security
 
-Brigade is opinionated about configuring projects and storing data like credentials.
-Because sensitive information is stored in a project's secret, care should be
-taken in preventing unauthorized access to that secret.
+Brigade is opinionated about configuring projects and storing data like
+credentials. Sensitive information relevant to a project should be set as
+[Secrets] on that project. In turn, project secrets are stored in a Kubernetes
+secret, so care should be taken in preventing unauthorized access to these
+resources.
 
-- Out-of-the-box, the project is the location where credentials should be stored
+- Out-of-the-box, credentials should be stored as project secrets
 - A project's credentials are accessible to any script running in that project,
   regardless of event.
-- For SSH-based Git clones, the SSH key should be stored on the project.
+- For SSH-based Git clones, the SSH key should be stored as a project secret.
 
-Note that if Helm is used to create a project, the project's secrets will be cached
-within Helm's release object. Read the [Helm docs](http://helm.sh) to learn how
-to secure Helm.
+[Secrets]: /topics/project-developers/secrets
 
 ## Script Security
 
-Brigade scripts can create pods, secrets, and persistent volume claims. Brigade does not
-evaluate the security of the containers that a pod runs. Consequently, it is best
-to avoid using untrusted containers in Brigade scripts. Likewise, it is not recommended
-to inject secrets into a container without first auditing the container.
+Brigade scripts can create pods, secrets, and persistent volume claims. Brigade
```

Comment:
I might suggest adding the word "indirectly" here so it's clear that scripts don't directly create job pods and other resources. The API server does all that.